docs/blog/2020-02-19-how-to-build-multilingual-sites-with-gatsby/index.md

```diff
@@ -115,54 +115,56 @@ If your website is not Google Maps, people most likely won't share their locatio
 
 As I mentioned already, the intranet app contains profile pages for all employees. They are generated dynamically because, well, the list of employees is also dynamic. This code piece that sits in `gatsby-node.js`'s `createPages` generates pages in the original implementation:
 
-```js
+```graphql
```

Comment:
Don't think we should split this one up. It should likely be kept together and use jsx as the language.

Reply:
changed